frontend/packages/client/src/App.js

```diff
@@ -30,15 +30,15 @@ function App() {
     >
       <QueryClientProvider client={queryClient}>
         <Web3Provider network={process.env.REACT_APP_FLOW_ENV}>
-          <NotificationModalProvider>
+          <Router>
             <NotificationServiceProvider>
-              <Router>
+              <NotificationModalProvider>
                 <ErrorHandler>
                   <AppPages />
                 </ErrorHandler>
-              </Router>
+              </NotificationModalProvider>
             </NotificationServiceProvider>
-          </NotificationModalProvider>
+          </Router>
```

Comment:
move `<Router>` to wrap the modal provider so that the modal can link out to other parts of the app